gst-plugin-pravega/src/pravegasink/imp.rs

```diff
@@ -1114,6 +1145,18 @@ impl BaseSinkImpl for PravegaSink {
             }
             *final_offset = Some(writer_offset_end);
 
+            if let Some(seconds) = simulate_failure_after_sec {
+                if !first_valid_time.is_none() {
+                    if (timestamp - first_valid_time.to_owned()).nanoseconds().unwrap() > (seconds as i32 * SECOND).nanoseconds().unwrap() {
+                        // TODO: close pravega writers
+                        //drop(writer.get_mut().get_mut().get_mut());
+                        //drop(index_writer);
+                        gst::element_error!(element, gst::CoreError::Failed, ["Simulate pravegasink failure"]);
```

Comment:
I don't see "Simulate pravegasink failure" in the test log. How can I confirm that this is actually simulating a failure?
Also, by not closing the writer, this is effectively performing a graceful stop as you can tell from these log messages:
```
0:00:35.579572114 50606 0x558c7e3ab860 INFO             pravegasink gst-plugin-pravega/src/pravegasink/imp.rs:1167:gstpravega::pravegasink::imp:<pravegasink0> stop: BEGIN
0:00:35.581769200 50606 0x558c7e3ab860 INFO             pravegasink gst-plugin-pravega/src/pravegasink/imp.rs:1221:gstpravega::pravegasink::imp:<pravegasink0> stop: Wrote final index record IndexRecord { timestamp: 2022-01-04T18:12:48.404852888Z (1641320005404852888 ns, 455922:13:25.404852888), offset: 163880, random_access: false, discontinuity: false }
0:00:35.583745207 50606 0x558c7e3ab860 INFO             pravegasink gst-plugin-pravega/src/pravegasink/imp.rs:1250:gstpravega::pravegasink::imp:<pravegasink0> stop: END: result=Ok(())
```

Reply:
log "Simulate pravegasink failure" in info.